concepts/numbers/introduction.md

```diff
@@ -0,0 +1,53 @@
+# Introduction
+The built-in number types in C++ can be divided into integers and floating points.
+Integers are whole numbers like `0`, `691`, or `-2`.
+Examples of floating point numbers are `6.02214076`, `0.1`, or `-1.616`.
+
+## Integers
+When you assign a value to an `int` variable, you can do so directly with a literal.
+There are different integer literals based on the base of the representation.
+Decimal integer literals are the most common and use the numbers `0` to `9`.
+By adding a special prefix it is possible to use other bases.
+The example below shows the number `9421` in its four representations and prefixes.
+
+```cpp
+int decimal{9241};              // base 10: 0-9
+int hexadecimal{0x24CD};        // base 16: 0-9 and A-F
+int binary{0b10010000011001};   // base  2: 0-1
+int octal{022031};              // base  8: 0-7
+                                // Leading with a 0 not the letter o.
+```
+You can use an apostrophe to separate digits for easier readability.
+`9'241` is the same as `0b0100'100'0001'1001` or `92'4'1`.
+
+## Floating-Point Numbers
```

Comment:
Suggestion: add a note along the lines of

> C++'s floats are IEEE 754 floats, just like in most other programming languages.
> These have finite precision (namely roughly such and such significant digits) and sometimes display weird behavior, such as `0.1 + 0.2 != 0.3` (link).

Reply:
I have some information in the about.md.
It's not so clean cut in C++. If you want a deep-dive:
"Unfortunately, C++ standard guarantees almost nothing about the built-in floating point types."
https://kkimdev.github.io/posts/2018/06/15/IEEE-754-Floating-Point-Type-in-C++.html